db/migrate/20180619065308_add_last_otp_at_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddLastOtpAtToUsers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :users, :last_otp_at, :datetime
+  end
+end
```

Comment:
Please merge these two migration. You can fix `ActiveRecord::Schema.define version:` manually.

Reply:
OK. I'll use the older version number.